<!-- TODO: Insert components of component-->
<launch>
  <arg name="role_name" default="hero" />
  <arg name="control_loop_rate" default="0.1" />

  <node pkg="perception" type="Position_Publisher_Node.py" name="Position_Publisher_Node" output="screen">
    <param name="control_loop_rate" value="0.1" />
    <param name="role_name" value="$(arg role_name)" />
  </node>

  <node pkg="perception" type="segmentation_node.py" name="SegmentationNode" output="screen">
    <param name="role_name" value="$(arg role_name)" />
    <param name="side" value="Center" />
  </node>
</launch>
